tests/unit_test/action/action_test.py

```
@@ -826,6 +826,136 @@ def test_prepare_request_no_request_params(self):
+            self,
+            mock_validate,
+            mock_get_media
+    ):
+
+        bot = "682323a603ec3be7dcaa75bc"
+
+        media_ids = ["media1"]
+
+        mock_get_media.side_effect = Exception("Media fetch failed")
+
+        with pytest.raises(Exception):
```

Ruff (0.15.5)

[warning] Do not assert blind exception: `Exception`

(B017)